### PR TITLE
Change appearance of reset member row

### PR DESCRIPTION
**Pull request by zanderz**
also clean up some naive/unneeded stuff I put in before, e.g. every row had a list of all team members (?)
Still might need some styling/tweaking but it pretty much looks like the designs.

### CODE REVIEW

File: shared/teams/team/members/container.js [hunk 1]
```diff
@@ -16,15 +16,26 @@ const getOrderedMemberArray = (
...
+        // both are inactive, compare usernames
+        return a.username.localeCompare(b.username)
+      }
+      // b is active, should go later
+      return -1
+    } else if (!b.active) {
+      // b is inactive, should come first
+      return 1
+    }
+    if (listYouFirst && you) {
```
Comment (chrisnojima) [inline shared/teams/team/members/container.js]: doesn't this listYouFirst logic trump everything? aka it should be above this active checking stuff

File: shared/teams/team/members/member-row/index.js [hunk 2]
```diff
@@ -45,20 +46,21 @@ export const TeamMemberRow = (props: Props) => {
     fullName,
     onChat,
     onClick,
-    type,
+    roleType,
     username,
```
Comment (chrisnojima) [inline shared/teams/team/members/member-row/index.js]: can we just change this to props: Props and not have very long destructuring. i think it makes it less readable and doesn't really serve a purpose

File: shared/teams/team/members/container.js (hunk 1 above)
Comment (zanderz) [reply shared/teams/team/members/container.js]: The ticket shows it this way in Zeppelin (reset user before self)

Comment (zanderz) [reply shared/teams/team/members/container.js]: also, per the design in the ticket, I added the chat button back in on the cancelled row

File: shared/teams/team/members/member-row/index.js [hunk 3]
```diff
@@ -77,56 +79,71 @@ export const TeamMemberRow = (props: Props) => {
...
-        <Avatar username={username} size={isMobile ? 48 : 32} />
-        <Box style={{...globalStyles.flexBoxColumn, marginLeft: globalMargins.small}}>
-          <Box style={globalStyles.flexBoxRow}>
-            <Usernames
-              type="BodySemibold"
-              colorFollowing={true}
-              users={[{username, following, you: you === username}]}
-            />
-            {!active && (
-              <Meta
```
Comment (zanderz) [inline shared/teams/team/members/member-row/index.js]: Also: android studio complained about this, so I changed it from Meta to Text

Comment (chrisnojima) [reply shared/teams/team/members/member-row/index.js]: Meta isn't text

**Pull request merged**